app/views/manage/users/show.html.haml

```diff
@@ -2,4 +2,30 @@
 
 %h1= @user.name
 %h2= @user.email
-  
\ No newline at end of file
+
+%h3 Your Indices:
+%table.table.table-condensed.table-striped
+  %thead
+    %tr
+      %th.col-sm-3 Community
+      %th.col-sm-2 AWRVI Index
+      %th.col-sm-2 Last Updated
+      %th.col-sm-3 Progress
+      %th.col-sm-2 View AWRVI Dashboard
+  %tbody
+    - @indices.each do |index|
+      %tr
+        %td
+          %b= index.community.name
+        %td
+          #{number_with_precision(index.vulnerability_index, precision: 2)}
+          - unless index.published_at.nil?
+            .label.label-warning Published
```

Comment:
I think we want to use label-primary to be consistent with the styling on the community show page.